### DISCUSSION TITLE
有时候发图没有任何响应，该如何debug呢

### DISCUSSION THREAD

**Discussion by Miuzarte** [Tsuk1ko/CQ-picfinder-robot#466] [问题求助]
如图，语料库都是正常的，error.log中最后一条在13:11:05，这时候开debug再--reload也没法复现
node v20.13.1
![f3226162570711445ece5123775f1650](https://github.com/Tsuk1ko/cq-picsearcher-bot/assets/66856838/03de6f13-79cb-4399-a172-63d612e35df3)

**Comment by Miuzarte** [comment]
直播推送和语料库一样也一直是正常的

lagrange的控制台也显示消息都在正常接收

**Comment by Tsuk1ko** [comment]
我也不知道了（
